subsys/usb/class/hid/core.c

```diff
@@ -54,16 +57,27 @@ USBD_CLASS_DESCR_DEFINE(primary) struct usb_hid_config hid_cfg = {
 			.wDescriptorLength = 0,
 		},
 	},
-	.if0_int_ep = {
+	.if0_int_in_ep = {
+		.bLength = sizeof(struct usb_ep_descriptor),
+		.bDescriptorType = USB_ENDPOINT_DESC,
+		.bEndpointAddress = CONFIG_HID_INT_IN_EP_ADDR,
+		.bmAttributes = USB_DC_EP_INTERRUPT,
+		.wMaxPacketSize =
+			sys_cpu_to_le16(CONFIG_HID_INTERRUPT_EP_MPS),
+		.bInterval = 0x09,
```

Comment:
Minor: Perhaps in a separate PR, we should add Interval configuration in Kconfig.